Mathlib/Data/ENNReal/Basic.lean

```diff
@@ -710,8 +710,28 @@ def evalENNRealOfNNReal : PositivityExt where eval {u α} _zα _pα e := do
     | _ => pure .none
   | _, _, _ => throwError "not ENNReal.ofNNReal"
 
+private lemma ennreal_one_pos : (0 : ℝ≥0∞) < 1 := zero_lt_one
+private lemma ofNat_pos {n : ℕ} [n.AtLeastTwo] : 0 < (no_index OfNat.ofNat n : ℝ≥0∞) :=
+  Nat.ofNat_pos
+
+/-- The `positivity` extension which identifies expressions of the form `OfNat.ofNat n : ℝ≥0∞`. -/
+@[positivity OfNat.ofNat _] def evalOfNatENNReal : PositivityExt where eval {u} α _z _p e := do
+  match u, α, e with
+  | 0, ~q(ℝ≥0∞), ~q(@OfNat.ofNat _ $n $instn) =>
+    try
+      let instn ← synthInstanceQ q(Nat.AtLeastTwo $n)
+      return Strictness.positive (q(@ofNat_pos $n $instn) : Expr)
+    catch _ => do
+      match n with
+      | ~q(1) => return .positive (q(ennreal_one_pos) : Expr)
+      | _ => throwError "not positive"
+  | _ => throwError "not `ENNReal`-valued `ofNat`"
+
 end Mathlib.Meta.Positivity
 
+example : (0 : ℝ≥0∞) < 1 := by positivity
+example : (0 : ℝ≥0∞) < 2 := by positivity
```

Comment:
I might add a test for \le or \ne as well, but I'll leave it to you whether that's useful

Reply:
No indeed, positivity already takes care of that